src/python/test/test_remote_datasets.py

```diff
@@ -0,0 +1,45 @@
+# This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+# See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+# Author(s):       Hind Montassif
+#
+# Copyright (C) 2021 Inria
+#
+# Modification(s):
+#   - YYYY/MM Author: Description of the modification
+
+
+from gudhi.datasets import remote
+import re
+
+def test_fetch_remote_datasets():
+    # Test files download from given urls
+    path_file_dw = remote.fetch("https://raw.githubusercontent.com/GUDHI/gudhi-data/main/points/spiral_2d.csv", "spiral_2d.csv")
+    names_dw = re.split(r' |/|\\', path_file_dw)
+    assert 'remote_datasets' == names_dw[0]
+    assert 'spiral_2d.csv' == names_dw[1]
```

Comment:
Those repetitive 3 checks could be performed in function.
We could also test in that function that `path_file_dw` is an existing file:
```python
import os.path
...
assert os.path.isfile(path_file_dw)
```

Reply:
Done.